### ISSUE TITLE
video time stamp?

### ISSUE THREAD

**Issue by cmkeeper9** [thecoder75/liveme-pro-tools#96]
Is there a way to get the video time stamp from a api query?  I have a deleted video which i have the id, but the urls now have date/time stamp.. obviously we know the date, but the time stamp? The video site still loads on a browser when i plug it in, but the video doesn't play.. it does have a play button though.

**Comment by thecoder75** [comment]
Yes, look in the ```liveme-pro-tools/src/electron/livemeapi.js``` file for the ```GetVideoInfo``` command.  This will return various details on the video itself including time stamp.  This command will only work if the video details haven't been purged out of the LiveMe database.  Once its been purged, there is no getting anything about it back.
